data/json/items/gun/22.json

```
@@ -379,5 +379,129 @@
         "item_restriction": [ "wp22mag" ]
       }
     ]
+  },
+  {
+    "id": "chiappabadger",
+    "looks_like": "rifle_22",
+    "type": "GUN",
+    "reload_noise_volume": 10,
+    "name": { "str_sp": "Chiappa Little Badger" },
+    "description": "A lightweight, ultra-compact, break-action folding rifle.  It is extremely basic, with a simple wire stock (which includes little 5 shot ammo holder) and a plain barrel.  Use to fold ",
+    "weight": "1315 g",
+    "volume": "1500 ml",
+    "longest_side": "926 mm",
+    "price": 23000,
+    "price_postapoc": 2000,
+    "to_hit": -1,
+    "bashing": 12,
+    "material": [ "steel", "plastic" ],
+    "symbol": "(",
+    "color": "brown",
+    "ammo": [ "22" ],
+    "skill": "rifle",
+    "ranged_damage": { "damage_type": "bullet", "amount": 2 },
+    "dispersion": 90,
+    "durability": 8,
+    "blackpowder_tolerance": 56,
+    "clip_size": 1,
+    "reload": 150,
+    "valid_mod_locations": [
+      [ "accessories", 1 ],
+      [ "barrel", 1 ],
+      [ "grip", 1 ],
+      [ "muzzle", 1 ],
+      [ "rail", 1 ],
+      [ "sights", 1 ],
+      [ "sling", 1 ]
+    ],
+    "faults": [ "fault_gun_blackpowder", "fault_gun_dirt" ],
+    "flags": [ "NEVER_JAMS", "RELOAD_EJECT" ],
+    "pocket_data": [ { "pocket_type": "MAGAZINE", "ammo_restriction": { "22": 1 } } ],
+    "use_action": { "target": "foldedchiappabadger", "msg": "You fold the smol rifle", "menu_text": "fold rifle", "type": "transform" }
+  },
+  {
+    "id": "foldedchiappabadger",
+    "type": "GUN",
+    "category": "weapons",
+    "name": { "str": "folded chiappa little badger" },
+    "description": "A folded chiappa badger 22LR rifle.  It is very compact in this form.  Use to unfold",
+    "weight": "1315 g",
+    "volume": "1000 ml",
+    "longest_side": "431 mm",
+    "price": 23000,
+    "price_postapoc": 2500,
+    "to_hit": -2,
+    "material": [ "steel", "plastic" ],
+    "symbol": "A",
+    "color": "brown",
+    "use_action": { "target": "chiappabadger", "msg": "You unfold the smol rifle", "menu_text": "unfold rifle", "type": "transform" }
+  },
+  {
+    "id": "ruger_1022takedown",
+    "looks_like": "ruger_1022",
+    "type": "GUN",
+    "reload_noise_volume": 10,
+    "name": { "str_sp": "Ruger 10/22 takedown" },
+    "description": "A ruger 10/22 version with the ability to be easily taken down into 2 parts for easier carrying.  It has polymer furniture and is also slightly lighter then the normal 10/22.  ",
```

```suggestion
    "description": "A ruger 10/22 version with the ability to be easily taken down into 2 parts for easier carrying.  It has polymer furniture and is also slightly lighter then the normal 10/22.",
```
No need for spaces at the end of description.